active/Expander/Expander.md

```diff
@@ -0,0 +1,155 @@
+
+# Background
+
+ > This spec corresponds to [this issue](https://github.com/microsoft/microsoft-ui-xaml/issues/3279) on the WinUI repo. 
+
+Throughout Windows, different expander controls are used by various apps and surfaces.
+There’s currently no consistent way to address this common UX pattern.
+This control is needed for situations where expanding (instead of overlaying) content is needed.
+An Expander control is motivated by its use in many app scenarios and supporting developers in migrating from
+[WPF](https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/expander-overview?view=netframeworkdesktop-4.8)
+and [Windows Community Toolkit](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/expander).  
+
+# API Pages
+
+## Expander class
+
+Represents the control that displays a header and a collapsible content area.
+
+> Spec note: the Expander class and ExpandDirection enum names match WPF.
+
+An Expander control is a UI component that provides a standard interaction for
+showing more content in a container that pushes adjacent content while expanding and collapsing.
+An Expander cannot be light dismissed and is independent of the contents inside it, including controls.
+Expander should be used when some content is only relevant some of the time (for example to
+read more information or access additional options for an item). 
+
+![A collapsed Expander that is expanded (pushing downwards) and then collapsed. The Header has the text "This is in the header" and the Content has the text "This is in the content". There is a TextBlock saying "This TextBlock is Below the Expander" that is pushed away](images/Expander_with_textblock.gif)
+
+### Examples
+
+#### Create an Expander
+
+![A collapsed Expander that is expanded and then collapsed. The Header has the text "This is in the header" and the Content has the text "This is in the content".](images/Expander.gif)
+
+XAML
+~~~~
+<muxc:Expander x:Name="DefaultExpander" 
+    Header="This text is in the header" 
+    Content="This is in the content"/>
+~~~~
+
+#### Put controls inside an Expander
+
+![A collapsed Expander. The Header has the text "This is in the header" and the Content has a button with the text "This ToggleButton is in the Content". The Expander is expanded, the button is toggled on and off, and the Expander is collapsed. ](images/Expander_togglebutton.gif)
+
+XAML
+~~~~
+<muxc:Expander x:Name="Expander2" Header="This is in the header"> 
```

Comment:
Presumably, I could also put content in the Header?

```xml
<muxc:Expander x:Name="Expander2">
    <muxc:Expander.Header>
        <Button>Button in the header</Button>
    </muxc:Expander.Header>
    <Button>Button in the content</Button>
</muxc:Expander>
```

Reply:
Yes

Reply:
To clarify this, I'll be updating the examples so the second one has a ToggleButton in the header as well!

Reply:
By this I mean putting a ToggleButton inside the Header as part of an example showing controls in the Header and Content.

Reply:
Some other interactive control would help. A toggle within a toggle creates some confusion.

Reply:
Recommend: Just some interactive content other than a ToggleButton.

Reply:
Maybe a TextBox or RichEditBox (a scenario might be "Enter your password" with the dropdown showing a "Forgot password button" or something like that)?

Reply:
![image](https://user-images.githubusercontent.com/12550607/107540086-8535a680-6b7a-11eb-9d8e-6b3c46eb94a6.png)

Reply:
Great idea @oldnewthing, that's a very good example; @kat-y I think we should use that.